doc/asciidoc/overview/configuration.adoc

```diff
@@ -0,0 +1,200 @@
+--
+
+Since the version `3.5.13` we introduced a new configuration systems based on
+dynamic reloading.
+
+=== Breaking Changes
+
+We deprecated the `neo4j.conf` file based configuration, so you need to define
+a new `streams.conf` file inside `$NEO4J_HOME/conf` and put in there all the
+required configuration.
+
+==== Note about usage with Docker
+
+The official Neo4j Docker image uses a particular naming convention for environment
+variables in order to transform them into properties inside the `neo4j.conf` file.
+In order to be compliant with that behaviour you can still use them without changing anything
+in your configuration, under-the-hood the new configuration system
+will save them inside the `streams.conf` file instead.
+
+=== How it behaves
+
+You can interact with the new configuration system in two ways:
+
+* By changing the `streams.conf` manually
+* By applying new configurations via procedures
+
+*You must consider that every change applied to the configuration causes
+the reload of the plugins so you must use this feature very carefully.*
+
+=== File based changes
+
+Every change inside the `streams.conf` is gathered and reloads the Streams `Sink`
+and `Source` with the new configuration.
+
+=== Procedure based changes
+
+With the new configuration system you'll find two new procedures:
+
+* `streams.configuration.get` returns the current configuration
+* `streams.configuration.set({<plugin_config_map>}, {<procedure_config>})` that applies
+the new configuration and returns it
+
+==== `streams.configuration.get`
+
+This procedure returns the current configuration applied to both `Sink` and `Source`
+plugin.
+
+Output Parameters:
+
+[cols="2*",options="header"]
+|===
+|Variable Name
+|Description
+
+|`name`
+|The configuration name
+
+|`value`
+|The configuration value
+
+|===
+
+So given the following procedure call:
+
+[source,cypher]
+----
+CALL streams.configuration.get() YIELD name, value
+RETURN name, value
+----
+
+You'll have this following output (it's related to *your* configuration 😄)
+
+[cols="2*",options="header"]
+|===
+|name
+|value
+
+|"streams.sink.topic.cypher.test"
+|"CREATE (p:Person{name: event.name, surname: event.surname})"
+
+|"streams.sink.errors.tolerance"
+|"all"
+
+|"kafka.default.api.timeout.ms"
+|"5000"
+
+|"kafka.bootstrap.servers"
+|"broker:9093"
+
+|"streams.sink.errors.log.include.messages"
+|"true"
+
+|"streams.sink.enabled"
+|"true"
+
+|"streams.sink.errors.log.enable"
+|"true"
+
+|"kafka.zookeeper.connect"
+|"zookeeper:2181"
+
+|===
+
+==== `streams.configuration.set`
+
+This procedure applies the map of configuration parameters passed as first argument
+and the procedure configuration parameters passed as second argument.
+
+Input Parameters:
+
+[cols="2*",options="header"]
+|===
+|Variable Name
+|Description
+
+|`plugin_config_map`
+|This map represents the set of configurations applied to the `Sink` and the `Source`
+
+|`procedure_config`
+|The configuration value
```

Comment:
Is there anything I can say in the second map other than save: true/false?   If this ties to a map in Apache Commons or something like that, then link it.

Reply:
At this moment we use only that propertyy